client/client.go

```diff
@@ -110,6 +110,12 @@ func ProxyUrl(pUrl string) Option {
 	}
 }
 
+func HttpClient(httpcl *http.Client) Option {
```

Comment:
Can you add some comments above noting what parameters will be ignored once this option is in.

Reply:
Done, Ty. Please check.